src/styles/views/packages/styles.less

```diff
@@ -10,4 +10,63 @@
       }
     }
   }
+
+  .selected-badge {
+
+    .badge {
+      margin-right: @base-spacing-unit * 1/4;
+    }
+  }
+}
+
+& when (@packages-view-enabled) and (@layout-screen-small-enabled) {
```

Comment:
I agree. I'd actually like to do away with the `& when (@{whatever}-enabled)` everywhere